openedx/core/djangoapps/discussions/models.py

```diff
@@ -32,40 +32,46 @@
     ['learn_more', 'configuration', 'general', 'accessibility', 'contact_email']
 )
 
+ProviderFeature = namedtuple('Feature', ['name', 'feature_support_type'])
```

Comment:
name it `ProviderFeature` as well. 

ProviderFeature = namedtuple('**ProviderFeature**', ['name', 'feature_support_type'])